eva_sub_cli/validators/validator.py

```diff
@@ -464,9 +464,9 @@ def _collect_md5sum_to_metadata(self):
         if md5sum_file:
             with open(md5sum_file) as open_file:
                 for line in open_file:
-                    sp_line = line.split()
+                    sp_line = line.split(' ')
                     md5sum = sp_line[0]
-                    vcf_file = sp_line[1]  # assuming no space in the vcf_file_name
+                    vcf_file = line.strip()[len(md5sum):].rstrip()  # Remove the md5: the rest is the file path
```

Comment:
```suggestion
                    vcf_file = line.strip()[len(md5sum):].lstrip()  # Remove the md5: the rest is the file path
```
(or just strip... otherwise there is a leading space)

Reply:
I meant to use `lstrip`. I guess I still don't know my left from my right ...